server/test/routes/articleRating.test.js

```diff
@@ -9,8 +9,8 @@ const author = {};
 const reader = {};
 const reader2 = {};
 const reader3 = {};
-const article = {};
-const article2 = {};
+let article = {};
+let article2 = {};
```

Comment:
i missed this before but rename `article2` to say `secondMockArticle` and `article` to `mockArticle`